docs/tutorial/flink.md

```diff
@@ -198,45 +191,105 @@ A [subset of the configuration schema](https://github.com/pangeo-forge/pangeo-fo
 that we just talked about related to inputs/outputs/caches gets dependency injected into the recipe by the runner. 
 
 Various other runner options ([documented here](https://pangeo-forge-runner.readthedocs.io/en/latest/reference/index.html) and [flink-specific here](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/deployment/config/)) 
-can also be passed either in the file configuration format above or directly during CLI `bake` calls. 
+can also be passed either in the file configuration format above or directly during CLI `bake` calls. It's recommended that you use
+the file configuration because some argument values are JSON based.
+
+Here's a quick example of a configuration file and `bake` command slightly more complicated where you're passing [flink-specific configuration options](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/deployment/config/) and runner options. 
+Note that `-f <path-to-your-runner-config>.<json||py>` would point to the runner configuration file talked about above:
+
+   ```python
+   import json
+   BUCKET_PREFIX = "s3://<bucket-name>/<some-prefix>/"
+   # The storage backend we want
+   s3_fsspec = "s3fs.S3FileSystem"
+   # Credentials for the backend
+   s3_args = {
+       "key": "<your-aws-access-key>",
+       "secret": "<your-aws-access-secret>",
+       "client_kwargs":{"region_name":"<your-aws-bucket-region>"}
+   }
+   # Take note: this is just python. We can reuse these values below
+   c.FlinkOperatorBakery.job_manager_resources = json.dumps({"memory": "2048m", "cpu": 1.0})
+   c.FlinkOperatorBakery.task_manager_resources = json.dumps({"memory": "2048m", "cpu": 1.0})
+   c.FlinkOperatorBakery.flink_configuration= json.dumps({
+      "taskmanager.numberOfTaskSlots": "1", 
+      "taskmanager.memory.flink.size": "1536m", 
+      "taskmanager.memory.task.off-heap.size": "256m", 
+      "taskmanager.memory.jvm-overhead.max": "1024m"
+   })
+   c.FlinkOperatorBakery.parallelism = 1
+   c.FlinkOperatorBakery.flink_version = "1.16"
+   
+   c.Bake.container_image = 'apache/beam_python3.9_sdk:2.50.0'
```

Comment:
This *must* match the version of python being used locally, as well as the version of beam used. Let's just add a comment here?

Reply:
done